### PR TITLE
step: fix version

### PR DESCRIPTION
**Pull request by dopey**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/step.rb [hunk 1]
```diff
@@ -20,12 +20,14 @@ class Step < Formula
   end
 
   def install
+    ENV["VERSION"] = "v0.15.15"
```
Comment (carlocab) [inline Formula/step.rb:23]:
```suggestion
    ENV["VERSION"] = version.to_s
```

File: Formula/step.rb [hunk 2]
```diff
@@ -20,12 +20,14 @@ class Step < Formula
...
     system "make", "build"
     bin.install "bin/step" => "step"
     bash_completion.install "autocomplete/bash_autocomplete" => "step"
     zsh_completion.install "autocomplete/zsh_autocomplete" => "_step"
 
     resource("certificates").stage do
+      ENV["VERSION"] = "v0.15.13"
```
Comment (carlocab) [inline Formula/step.rb:30]:
```suggestion
      ENV["VERSION"] = resource("certificates").version.to_s
```

**Review by carlocab — CHANGES_REQUESTED** [review]
Thanks, @dopey.

Please bump the `revision` (either add `revision 1` or increment the `revision` right after `license`) so that users get the updated version of the formula when they do `brew upgrade`. 

Squash this change with your first commit and use the message
```
step: fix version
```
(or similar). We use messages like `step 0.15.15` for version bump commits.

Thanks!

File: Formula/step.rb (hunk 2 above)
Comment (Bo98) [reply Formula/step.rb:30]: Perhaps even make it:

```rb
    resource("certificates").stage do |r|
      ENV["VERSION"] = r.version.to_s
```

**Comment by dopey** [comment]
Updated. Thanks for the tips!

**Review by carlocab — APPROVED** [review]

**Pull request closed without merging**